tracardi/process_engine/action/v1/google_translator_action.py

```diff
@@ -0,0 +1,102 @@
+from pydantic import validator
+
+from tracardi.service.plugin.domain.config import PluginConfig
+
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormField, FormComponent
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.service.plugin.domain.result import Result
+
+from googletrans import Translator
+
+
+class Config(PluginConfig):
+    text_to_translate: str
+    source_language: str
+
+    @validator("source_language")
+    def check_given_source_language_type(cls, value):
+        if not isinstance(value, str) and value != "":
+            raise ValueError("Give language source must be a string!")
+        return value
+
+
+def validate(config: dict) -> Config:
+    return Config(**config)
+
+
+class GoogleTranslateAction(ActionRunner):
+    config: Config
+    translator: Translator
+
+    async def set_up(self, init):
+        self.config = validate(init)
+        self.translator = Translator()
+
+    async def run(self, payload: dict, in_edge=None) -> Result:
+        translation = self.translator.translate(self.config.text_to_translate, src=self.config.source_language,
```

Comment:
well, this way you do not support dot paths. User can hard-code some text, but nothing more. Text should be accessed with DotAccessor instance.